gen/elerning_mu_ff

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use File::Basename;
+
+our $SERVICE_NAME = basename($0);
+our $PROTOCOL_VERSION = "3.0.1";
+my $SCRIPT_VERSION = "3.0.1";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+my $data = perunServicesInit::getHashedHierarchicalData;
+
+#Constants
+our $A_USER_LOGIN;                   *A_USER_LOGIN =                     \'urn:perun:user:attribute-def:def:login-namespace:mu';
+our $A_USER_FIRST_NAME;              *A_USER_FIRST_NAME =                \'urn:perun:user:attribute-def:core:firstName';
+our $A_USER_LAST_NAME;               *A_USER_LAST_NAME =                 \'urn:perun:user:attribute-def:core:lastName';
+our $A_USER_STATUS;                  *A_USER_STATUS =                    \'urn:perun:member:attribute-def:core:status';
+
+our $STATUS_VALID;                   *STATUS_VALID =                     \'VALID';
+
+
+my $fileName = "$DIRECTORY/$::SERVICE_NAME";
+
+my $users;
+
+my %facilityAttributes = attributesToHash $data->getAttributes;
```

Comment:
Whole processing of `$data` has to be updated to work with the new hashed data, since the returned JSON structure is different. See reworked scripts and do the similar like in #590.

Also please update script version afterwards, eg. `3.0.1` -> `3.0.2`.